iulianIachimov/src/main/java/datacollectiontask/Main.java

```diff
@@ -0,0 +1,123 @@
+
+        Map<String, Address> addressMapOne = new HashMap();
+        addressMapOne.put("Home", homeAddressOne);
+        addressMapOne.put("Work", workAddressOne);
+
+        //2
+        Map<String, Address> addressMapTwo = new HashMap();
+        addressMapTwo.put("Home", new Address("Republic of Moldova", "Chisinau", "12 Dacia Str."));
+        addressMapTwo.put("Work", new Address("Republic of Moldova", "Chisinau", "45/3 Hotin Str."));
+
+        //Creating users
+        User userOne = new User("Jill Jackson", addressMapOne);
+        User userTwo = new User("Adam Smith", addressMapTwo);
+
+        // create first list of items
+        List<Item> ItemListOne = new ArrayList<>();
+        Item itemOne = new Item("Iphone", Type.GADGET);  // create itemOne
+        Item itemTwo = new Item("Trousers", Type.CLOTHES); // create itemTwo
+        ItemListOne.add(itemOne); // add itemOne to ItemListOne
+        ItemListOne.add(itemTwo);
+
+        //Create second list of items
+        List<Item> ItemListTwo = new ArrayList<>();
+        Item itemThree = new Item("Trotinette", Type.TOY);  // create itemOne
+        Item itemFour = new Item("Frying Pan", Type.KITCHENWARE); // create itemTwo
+        ItemListTwo.add(itemThree); // add itemOne to ItemListOne
+        ItemListTwo.add(itemFour);
+
+        //Create 2 objects of Order type ()
+        Order orderOne = new Order(userOne, ItemListOne);
+        Order orderTwo = new Order(userTwo, ItemListTwo);
+
+        //Create list of orders
+        List<Order> orderList = new ArrayList<>();
+        orderList.add(orderOne);
+        orderList.add(orderTwo);
+
+        //Filter Orders by Type CLOTHES, calling funcion filterOrderByItemType(orderList)
+        List<Order> filteredOrderList = filterOrderByItemType(orderList);
+        System.out.println("the size of filtered order list is " + filteredOrderList.size());
+
+        /*
+        //collection sort, out of tasks
+        System.out.println("userList before sorting");
+        for (User user: userList){
+            System.out.println(user.username);
+        }
+
+        Collections.sort(userList, new Comparator<User>() {
+            @Override
+            public int compare(User userOne, User userTwo) {
+                return userOne.username.compareTo(userTwo.username);
+            }
+        });
+
+        System.out.println("userList after sorting");
+        for (User user: userList){
+            System.out.println(user.username);
+        }
+    */
+
+
+        // Creating the list for users
+        List<User> userList = new ArrayList<>();
+        userList.add(userOne);
+        userList.add(userTwo);
+
+        // Filter users by city
+        List<User> filteredListofUsers = new ArrayList<>();
+        filteredListofUsers = filterUsersByCity(userList, "Chisinau"); // call for viewing the users from Chisinau city
+        System.out.println("The filtered list of users is " + filteredListofUsers.size());
```

Comment:
The filtred list of users is 1 ? I think that there should be displayed not size, but the user properties. Could you please correct it?

Reply:
I guess it is a bit redundant as I have returned the user properties within the function itself. Lt me know if I can just commit by simply deleting 
System.out.println("The filtered list of users is " + filteredListofUsers.size());